appveyor.yml

```diff
@@ -128,7 +128,7 @@ after_build:
       # This packages via CMakeList CPack settings (Tar.Gzip for SW upload)
       # msbuild $env:MSBuildOptions PACKAGE.vcxproj
 
-      7z a -ttar -y -bd nupic_core-$env:APPVEYOR_REPO_COMMIT-windows64.tar .
+      7z a -ttar -y -bd nupic_core-$env:APPVEYOR_REPO_COMMIT-windows64.tar . | findstr /b /c:"Everything is Ok" /c:"Scanning" /c:"Creating archive"
 
```

Comment:
This reduces the CI build log output considerably, all the "Compressing ..." lines.